.github/workflows/compute.yml

```
@@ -0,0 +1,52 @@
+name: "@ubiquity-os-marketplace/command-ask"
+
+on:
+  workflow_dispatch:
+    inputs:
+      stateId:
+        description: "State Id"
+      eventName:
+        description: "Event Name"
+      eventPayload:
+        description: "Event Payload"
+      settings:
+        description: "Settings"
+      authToken:
+        description: "Auth Token"
+      ref:
+        description: "Ref"
+      signature:
+        description: "Signature sent from the Kernel"
+
+jobs:
+  compute:
+    name: "LLM Reply"
+    runs-on: ubuntu-latest
+    permissions: write-all
+    env:
+      SUPABASE_URL: ${{ secrets.SUPABASE_URL }}
+      SUPABASE_KEY: ${{ secrets.SUPABASE_KEY }}
+      VOYAGEAI_API_KEY: ${{secrets.VOYAGEAI_API_KEY}}
```

```suggestion
      VOYAGEAI_API_KEY: ${{ secrets.VOYAGEAI_API_KEY }}
```
(applies to all the secrets below)